Signal/src/Models/TSMessageAdapaters/TSPhotoAdapter.m

```diff
@@ -50,6 +50,10 @@ - (UIView *)mediaView {
         imageView.frame         = CGRectMake(0.0f, 0.0f, size.width, size.height);
         imageView.contentMode   = UIViewContentModeScaleAspectFill;
         imageView.clipsToBounds = YES;
+        // Use trilinear filters for better scaling quality at
+        // some performance cost.
+        imageView.layer.minificationFilter = kCAFilterTrilinear;
+        imageView.layer.magnificationFilter = kCAFilterTrilinear;
```

Comment:
I've mixed feelings about using higher quality scaling in the messages view.  On the one hand, images will be usually scaled down quite a bit and thus vulnerable to scaling artifacts.  On the other hand, this could hypothetically affect scrolling performance.  I don't think it will though; Signal conversations aren't likely to have more than a handful of still images in view at any given time.